Backend/tests/test_user_ctrl.py

```diff
@@ -327,6 +327,20 @@ def test_joinClass(self):
     #confirm none joined
     self.assertFalse(joined)
 
+  def test_resetPassword(self):
```

Comment:
I wrote more tests to cover more, but the function that actually sends the email cannot be tested for obvious reasons